src/messaging-component.js

```diff
@@ -52,7 +52,7 @@ const showExitModal = ({ cartRecovery }) => {
   return isRendered;
 };
 
-export const Messaging = (...args : $ReadOnlyArray<{ cartRecovery : { userId : string, beta? : boolean } }>) => {
+export const Messaging = (...args : $ReadOnlyArray<{| cartRecovery : {| userId : string, beta? : boolean |} |}>) => {
```

Comment:
If someone has already integrated with this and are sending more properties in will this break their integration? Also why do we want to change this? Just to be explicit?

Reply:
the change id one by running `flow-typed` I can revert them, I would need to check if this is justa. new flow syntax change

Reply:
you're right the change makes it exact match https://flow.org/en/docs/types/objects
I'll revert it to make it safe

Reply:
good catch